src/insurancesCalculator.js

```diff
@@ -35,51 +35,56 @@ const POLICY_UPDATES = [
  * @param {Number} gross 
  * @param {1|2|3|4} region 
  * @param {Date} date 
- * @returns { {
- * total: number,
- * details: [{name: string, amount: number}]
-* }}
+ * @returns {Promise<{total: number, details: { name: string, amount: number }[]}>}
```

Comment:
Wrong property name. It should be `insurances` instead of  `details`. See the return obj in last line